### PR TITLE
Typo fix and release flag

### PR DESCRIPTION
**Pull request by russ**
Fixing a typo in the guide and adding a `--release` flag to the luck_cli build reference.

### CODE REVIEW

File: src/actions/guides/getting-started/installing.cr [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Guides::GettingStarted::Installing < GuideAction
     * cd into the newly cloned directory
     * Check out latest [released version](https://github.com/luckyframework/lucky_cli/releases) `git checkout #{LuckyCliVersion.current_tag}`
     * Run `shards install`
-    * Run `crystal build src/lucky.cr`
+    * Run `crystal build --release src/lucky.cr`
```
Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]: @paulcsmith I thought this should be here, but what's your thoughts on this? Not sure if there's a reason to have this in a non-release build.

Comment (paulcsmith) [reply src/actions/guides/getting-started/installing.cr]: The main reason was that it takes forever to build in release mode and the CLI does so little that the performance optimizations aren't noticeable. So I think we can keep it as it was

Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]:
```suggestion
    * Run `crystal build src/lucky.cr`
```

**Review by jwoertink — APPROVED** [review]

File: src/actions/guides/getting-started/installing.cr (hunk 1 above)
Comment (jwoertink) [reply src/actions/guides/getting-started/installing.cr]: OMG. github... Apparently I can't commit that suggestion, and it keeps throwing an error when I try to unapprove 😝

**Review by jwoertink — APPROVED** [review]

**Pull request merged**